cx_Freeze/hooks.py

```diff
@@ -814,3 +815,12 @@ def load_clr(finder, module):
     module_dir = os.path.dirname(module.file)
     dllname = 'Python.Runtime.dll'
     finder.IncludeFiles(os.path.join(module_dir, dllname), dllname)
+    
+    
+def load_sqlite3(finder, module):
+    """In Windows, the sqlite3 module requires an additional dll sqlite3.dll to
+       be present in the build directory."""
+    if sys.platform == "win32":
+        dll_name = "sqlite3.dll"
+        dll_path = os.path.join(sys.base_prefix, "DLLs", dll_name)
```

Comment:
`sys.base_prefix` breaks on Python 2.7, since `sys.base_prefix` was only added in Python 3.3 ([see docs](https://docs.python.org/3/library/sys.html#sys.base_prefix)). In older Python installations, one should look for `sys.real_prefix` which is set by `virtualenv` within virtual environments, and fall back to `sys.prefix` which is always set (but does not point to the system installation if run from a virtual environment).

See pull request #272.